src/index.js

```diff
@@ -98,12 +99,14 @@ exports.resolve = (source, file, opts) => {
         cwd: options.cwd || projectRootDir,
         root: options.root || [],
         alias: options.alias || {},
+        resolvePath: options.resolvePath,
```

Comment:
Shouldn't this be `resolvePath: options.resolvePath || resolvePath`?
So that you can remove your line 108 and always get the `src` from `pluginOptions.resolvePath`?

Reply:
> Shouldn't this be `resolvePath: options.resolvePath || resolvePath`?
> So that you can remove your line 108 and always get the `src` from `pluginOptions.resolvePath`?

I thought about it. It's similar, but due to the importance of resolvePath function, I made the fallback assignment right before it's being used, so if there's a falsy value coming from options or plugin, the module still works.

Reply:
> so if there's a falsy value coming from options or plugin, the module still works.

In theory, this should really never happen since `resolvePath` will be the first default value being set. So any falsy value will always result to using the default function.

But I see your point to make extra sure (especially without using typescript here), so I'll accept it ;)